packages/api/cms-api/package.json

```diff
@@ -79,7 +79,7 @@
         "uuid-by-string": "^4.0.0"
     },
     "devDependencies": {
-        "@aws-sdk/types": "3.734.0",
+        "@aws-sdk/types": "^3.734.0",
```

Comment:
There are `dependenciesComments` at the end of the file describing while this pin is necessary. Please also remove those.